hardware/i2c.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+order:
+title: i2c
+heading: Schematics
+description: micro:bit i2c pins
+permalink: /hardware/schematic/
+ref: i2c
+lang: en
+assigned-to: markw
+review-with: jonnya
+---
+
+If you make an accessory for the micro:bit, please help us by editing the page and sharing the details of the i2c addresses you use.
+
+## Use of the shared I2C bus
+
+The motion sensors on the board are on the same I2C bus as the edge connector I2C pins. This means that if you have an accessory that uses I2C on this bus, you need to check it won’t clash with the new sensors.
+
+Because we want to avoid having to do another revision in the future, we are designing in two possible sensor devices, but all micro:bits will ship with only one combined magnetometer and accelerometer. The DAL will automatically select which driver to instantiate (and hence this will work in MicroPython and MakeCode without any issues).
+
+## Table of addresses used
+
+|                     | accelerometer    | magnetometer (compass) |
+|---------------------|------------------|------------------------|
+| micro:bit v1.3      | 0x1D (0x3A/0x3B) | 0x0E (0x1C/0x1D)       |
+| micro:bit variant 1 | 0x19 (0x32/0x33) | 0x1E (0x3C/0x3D)       |
```

Comment:
I think we're probably good to call out that variant 1 is LSM303AGR and variant2 is FXOS8700CQ now we've also mentioned that on the hardware page. Let's stick it in brackets after the name:

|                     | accelerometer    | magnetometer (compass) |
|---------------------|------------------|------------------------|
| micro:bit v1.3 (MMA8653+MAG3110)   | 0x1D (0x3A/0x3B) | 0x0E (0x1C/0x1D)       |
| micro:bit variant 1 (LSM303AGR) | 0x19 (0x32/0x33) | 0x1E (0x3C/0x3D)       |
| micro:bit variant 2 (FXOS8700CQ)| 0x1E (0x3C/0x3D) | 0x1E (0x3C/0x3D)       |

Reply:
ack done